xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -427,6 +427,17 @@
       class="suggest-propertyValues" multiple="multiple" size="1"
       data-className="$!escapetool.xml($xproperty.className)" data-propertyName="$!escapetool.xml($xproperty.name)">
     </select>
+  #elseif ($filterType == 'page')
+    #set ($discard = $xwiki.linkx.use($services.webjars.url('selectize.js', 'css/selectize.bootstrap3.css'),
+      {'type': 'text/css', 'rel': 'stylesheet'}))
+    #set ($discard = $xwiki.ssfx.use('uicomponents/suggest/xwiki.selectize.css', true))
+    #set ($discard = $xwiki.jsfx.use('uicomponents/suggest/suggestPages.js',
+      {'forceSkinAction' : true, 'language' : $xcontext.locale}))
+    <select id="xwiki-livetable-${htmlLiveTableId}-filter-$velocityCount" name="$!escapetool.xml($column)"
+      #if ("$!columnProperties.size" != '')size="$!escapetool.xml($columnProperties.size)"#end
+      #if ("$!columnProperties.multiple" != '')multiple="$!escapetool.xml($columnProperties.multiple)"#end
```

Comment:
The ``multiple`` property is not listed on https://extensions.xwiki.org/xwiki/bin/view/Extension/Livetable+Macro#HParameter24columnsProperties so we didn't need it before. I suppose it could be useful, although I think the users will want to be able to select multiple values, most of the time. The question is what value do you set? Because the [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#Attributes) says:

> When multiple is specified, most browsers will show a scrolling list box instead of a single line dropdown.

So it's enough to be specified, no matter the value. Thus if someone sets ``$columnProperties.multiple`` to ``false`` it will still be specified.

Reply:
I thought about that too and I was thinking that it should be the user responsibility to set the value he wants. If, as you said, someone sets `$columnProperties.multiple` to `false`, he should see right away that "multiple" is in fact enabled. What I mean is that I think it is as much error prone as just setting the `multiple` property in the html tag.